itest/lnd_quiescence_test.go

```diff
@@ -60,4 +67,40 @@ func testQuiescence(ht *lntest.HarnessTest) {
 		alice, req,
 		lnrpc.PaymentFailureReason_FAILURE_REASON_INSUFFICIENT_BALANCE,
 	)
+
+	// Alice now restarts with an extremely short quiescence timeout.
+	ht.RestartNodeWithExtraArgs(
+		alice, []string{"--htlcswitch.quiescencetimeout=1ms"},
+	)
+
+	// Once restarted, the channel is no longer quiescent so Alice can
+	// finish the payment for invoice2.
+	ht.SendPaymentAssertSettled(alice, req)
+
+	// Bob adds another invoice.
+	invoice3 := bob.RPC.AddInvoice(invReq)
+
+	// Alice now requires the channel to be quiescent again. Since we are
+	// using a short timeout (1ms) for the quiescence, Alice should
+	// disconnect from Bob immediately.
+	res = alice.RPC.Quiesce(&devrpc.QuiescenceRequest{
+		ChanId: chanPoint,
+	})
+	require.True(ht, res.Initiator)
+
+	// The above quiescence timeout will cause Alice to disconnect with Bob.
+	// However, since the connection has an open channel, Alice and Bob will
+	// be reconnected shortly.
+	//
+	// TODO(yy): When refactoring the peer connection we need to have an RPC
+	// that reports how many times the peer has been connected.
+	ht.AssertConnected(alice, bob)
```

Comment:
I think we can this this call here? https://lightning.engineering/api-docs/api/lnd/lightning/subscribe-peer-events/